components/kyma-environment-broker/common/hyperscaler/account_pool.go

```diff
@@ -27,161 +26,136 @@ type Credentials struct {
 }
```

Comment:
please move `Credentials` type to the `account_provider` file

Reply:
Done!